logstash-core/lib/logstash/plugin.rb

```diff
@@ -46,8 +45,8 @@ def eql?(other)
   end
 
   def initialize(params=nil)
+    @logger = self.class.logger
```

Comment:
how can a mixin provide an instance variable? the mixin is included in the class once... not for every instance. unless I am missing something?